app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,4 @@
+nav
+  ul.pager
```

Comment:
This is supposed to be "pagination" class, see Bootstrap docs. Never mind, fix it in your next PR when you fix the nav bar.